src/slab.py

```diff
@@ -0,0 +1,54 @@
+"""Slab data stucture that is used to represent Order book."""
+from construct import Int8ul, Int32ul, Int64ul, PaddedString, Padding  # type: ignore
+from construct import Struct as cStruct
+from construct import Switch
+
+KEY = cStruct(
```

Comment:
Construct's default byte order is big-endian. Not sure if you're supposed to set a swap param here, but you need to make sure we serialize to little-endian byte order.

Reply:
Good point.